### ISSUE TITLE
FR or Q? Can the repo be deployed into multiple times into multiple 'parent_folder' in the same org

### ISSUE THREAD

**Issue by rr4444** [terraform-google-modules/terraform-example-foundation#803]
### TL;DR

Can the repo be deployed multiple times into multiple 'parent_folder' in the same org?

My suspicion is that it cannot, because the policies are applied org-wide, rather than folder specific. Multiple repo deployments would just overwrite each other's policies, etc. Is that assumption correct? So maybe or would be 'fine', as later repo deployments would merely fail at that stage (which isn't a problem if org policies should be the same for each deployment anyway?).

If there are instead no policy problems, are there other reasons not to deploy the repo into multiple folders?

Many thanks!
